mathesar/models.py

```diff
@@ -18,13 +18,17 @@ def __str__(self):
         return f"{self.__class__.__name__}: {self.name}"
 
 
+class Schema(DatabaseObject):
+    database = models.CharField(max_length=63)
```

Comment:
We may want more characters here to handle multiple hosts (in the case where the user-defined tables are on a separate host).